macos/ONLYOFFICE/update/updates/ONLYOFFICE/changes/8.1.0/changes.html

```diff
@@ -91,46 +91,80 @@ <h2>New Features</h2>
 
         <h4>All Editors</h4>
         <ul>
-            <li>TODO</li>
+            <li>Switching to the creating forms in the <code>PDF</code> format</li>
+            <li>Added the ability to disable the 'Connect to cloud' section via the <code>--lock-portals</code> key</li>
```

Comment:
<li>Added the ability to disable the <code>Connect to cloud</code> section via the <code>--lock-portals</code> key</li>